docs/mine-hnt/light-hotspots/guides/kerlink.mdx

```diff
@@ -0,0 +1,251 @@
+```
+or directly from the gateway:
+```
+root@klk-wiis-050029:~ # wget https://github.com/helium/gateway-rs/releases/download/v1.0.0-alpha.8/helium-gateway-v1.0.0-alpha.8-klkgw.ipk
+Connecting to github.com (192.30.255.112:443)
+Connecting to github-releases.githubusercontent.com (185.199.110.154:443)
+helium-gateway-v1.0. 100% |*************************************************************************************************************|  1032k  0:00:00 ETA
+root@klk-wiis-050029:~ # mv helium-gateway-v1.0.0-alpha.8-klkgw.ipk /user/.updates
+root@klk-wiis-050029:~ # sync; kerosd -u; reboot; exit
+
+The system is going down for reboot NOW!029 (pts/0) (Tue Apr 20 17:49:37 2021)
+```
+
+This is used because KerOS uses a backup mechanism when installing packages. As soon as you install an IPK, a backup snapshot is made. You can then
+
+* restore to stock firmware: `kerosd -s`
+* restore to the latest backup: `kerosd -b`
+* trigger an update from `/user/.updates`: `kerosd -u`
+
+## Check installation
+
+You need the latest KerOS version (see above) and the appropriate light miner software installed:
+
+```
+root@klk-wiis-050029:~ # opkg list
+helium_gateway - 1.0.0-alpha.8
+keros - 4.3.3-0-g993efce1
+```
+
+## Configure LoRa Frequency Plan
+
+Use the built-in tool `klk_apps_config` to activate the Kerlink Packet Forwarder (CPF) on the appropriate region:
+
+```
+# this is multiline just for readability
+root@klk-wiis-050029:~ # klk_apps_config --activate-cpf \
+    --lns-server 127.0.0.1 \
+    --lns-dport 1680 --lns-uport 1680 \
+    --loradconf US915-US.json
+Stopping lorad 1.3.0
+Stopping lorafwd 1.2.0
+Starting lorad 1.3.0
+Starting lorafwd 1.2.0
+lorafwd[1447]: <6> Parsing configuration file: /var/run/lora/gateway-id.toml
+lorafwd[1447]: <6> Parsing configuration file: /etc/lorafwd.toml
+root@klk-wiis-050029:~ #
+```
+
+The list of available preconfigured regions are in `/etc/lorad/XXX` where XXX is the gateway model nickname:
+* iFemtoCell: `wifc`
+* iFemtoCell Evolution: `fevo`
+* iStation: `wiis`
+
+```
+root@klk-wiis-050029:~ # ll /etc/lorad/wiis/
+drwxr-xr-x    2 root     root         220 Aug  3  2020 .
+drwxr-xr-x    1 root     root        4.0K Apr 20 17:55 ..
+-rw-r--r--    1 root     root        2.4K Aug  3  2020 AS923-JP.json
+-rw-r--r--    1 root     root        2.0K Aug  3  2020 AS923-TH-SG-HK-TW.json
+-rw-r--r--    1 root     root        1.9K Aug  3  2020 AU915-AU.json
+-rw-r--r--    1 root     root        2.0K Aug  3  2020 EU868-FR.json
+-rw-r--r--    1 root     root        1.8K Aug  3  2020 IN865-IN.json
+-rw-r--r--    1 root     root        2.3K Aug  3  2020 KR920-KR.json
+-rw-r--r--    1 root     root        1.9K Aug  3  2020 RU864-RU.json
+-rw-r--r--    1 root     root        1.9K Aug  3  2020 RU864-RU_Legacy.json
+-rw-r--r--    1 root     root        1.9K Aug  3  2020 US915-US.json
+```
+
+If the desired configuration file is not available, feel free to create a new one and restart the tool.
+
+## US915/AU915
```

Comment:
Are those steps not needed for other regions? In the next paragraph, it is indeed stated as such. It's OK in that case. If such a step is needed for other regions, I find the current structure confusing. Thanks!

Reply:
no, this step is only required for US915 and AU915